AZURE_DEVELOPER.md

```diff
@@ -100,4 +100,12 @@ Generating schema definition is simple in Ansible since we only need to call `to
 
 Marshalling is another tough job to do because Azure SDK objects are deeply hierarchical objects. Both Ansible and Terraform handle it in a recursive way (recursive code template). We put both-way marshalling code templates in `templates/azure/[terraform|ansible]/sdktypes` folder. You will be able to find the corresponding helper functions by using the names of the templates.
 
-It is not too difficult to generate Azure SDK API calls since they are all defined in `azure_sdk_definitions` section of `api.yaml`. Typically API calls are put directly in the resource/module code template, but with an reusable method call sub-template.
\ No newline at end of file
+It is not too difficult to generate Azure SDK API calls since they are all defined in `azure_sdk_definitions` section of `api.yaml`. Typically API calls are put directly in the resource/module code template, but with an reusable method call sub-template.
+
+## Command Line Option
+
+Besides the basic command line options, we add the `-c (--cloud)` option for users to specify the target cloud platform. The supported values are:
+* “gcp”： Google Cloud Platform (by default).
```

Comment:
Please do not use Chinese characters (：).

```suggestion
* `gcp`: Google Cloud Platform (by default).
```

Reply:
Modified.